audio/src/fetch.rs

```diff
@@ -1,23 +1,30 @@
 use crate::range_set::{Range, RangeSet};
 use byteorder::{BigEndian, ByteOrder, WriteBytesExt};
 use bytes::Bytes;
-use futures::sync::{mpsc, oneshot};
-use futures::Stream;
-use futures::{Async, Future, Poll};
 use std::cmp::{max, min};
 use std::fs;
```

Comment:
I noticed this code uses `std::fs` which is the blocking file api. Is the intention to create an async audio file type? There is `tokio::fs` which is an async `File` type. Now, it's not very commonly used and async file io in rust isn't very good at the moment, but would using that reduce some of the logic that's needed in here?

Reply:
Interesting point - but won't this task be network bound anyway? So would it be a big improvement to use [`tokio::fs::write`](https://docs.rs/tokio/0.2.21/tokio/fs/fn.write.html)?